dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.passwordUtilities1.1.internal.ee-6.0.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.passwordUtilities1.1.internal.ee-6.0
```

Comment:
With it being a new feature we have the opportunity to change to have a internal.ee-8.0 with appSecurity-3.0, servlet 4.0 and transaction 1.2 and then take servlet-4.0 out of the auto feature.  This will be a cleaner solution for Java EE 8 because then we are forcing them on the EE 8 only features and not tolerating appsecurity 2.0 and 1.0 at all for the EE 8 features.  For this feature you would still have 2.0 tolerate 1.0 for appSecurity and 3.0 tolerate 3.1 for servlet and transaction 1.1 tolerates 1.2 for transaction.

Reply:
Made the suggested changes.